src/components/buttonsIndicators/button/index.tsx

```diff
@@ -22,6 +22,8 @@ export interface Props {
   id?: string
   disabled?: boolean
   icon?: {image: React.ReactNode, position: 'before' | 'after'}
+  beforeIcon?: React.ReactNode,
```

Comment:
does the existing code not work for your case? in `icon` object you already have `before` and `after` options

Reply:
I need both `before` and `after` at the same time, which is why I changed this.

Reply:
could we use existing icon object to achieve this and just extend it? I would like to avoid duplicating props.

Reply:
I couldn't think of a good way to extend the current `icon` prop to work for two different icons, so I thought this would be the cleanest solution. But if you have a good idea for how to do it inside `icon` that totally works.

Reply:
cc @petemill for his thoughts

Reply:
This might need some design decision to clarify the change to the button in the system - @rossmoody? I'm not sure it's easily understandable for the user to grasp the meaning of the button when there are 2 icons.
![image](https://user-images.githubusercontent.com/741836/59630292-d4071080-90f9-11e9-895a-45648abc52b1.png)

If it isn't a button, but is some kind of 'status chip' then let's make a new component called that. It could for example have a prop which determines which icon is displayed called `status` which accepts `'success' / 'fail' / 'info'`

Reply:
This is the first I'm seeing this and sadly I really don't have any insight as to how this works or what function it serves. @jenn-rhim will need to chime in on details surrounding this.

Reply:
This is not the approved design. We removed this state. @NejcZdovc

Reply:
@petemill @rossmoody @jenn-rhim so even though that that specific button was not needed we still need it for this one:

![image](https://user-images.githubusercontent.com/9574457/61465787-0619c580-a979-11e9-83aa-ca621aff293e.png)